glide.yaml

```diff
@@ -1,38 +1,29 @@
 package: github.com/projectcalico/calico-bgp-daemon
 import:
 - package: github.com/Sirupsen/logrus
+  version: ^0.11.2
 - package: github.com/coreos/etcd
-  version: ~3.1.0-alpha.1
+  version: ^3.1.1
   subpackages:
   - client
   - pkg/transport
 - package: github.com/osrg/gobgp
-  version: ee8ce99e2df1f732ef515e20fb3054c0cff0249b
+  version: ^1.16.0
   subpackages:
   - api
   - config
   - packet/bgp
   - server
   - table
 - package: github.com/projectcalico/libcalico-go
-  version: v1.0.2
+  version: ^1.1.0
```

Comment:
- This is missing a `v` in front of `1.1.0` :) (seems like glide is smart enough to figure that out though) 
- Also, I'm not a big fan of `^` (this version of higher available) in glide.yaml because it would update it to a newer version when you're adding a new dependency, as a side effect. Don't have a strong preference though, if you think using `^` is fine